@ecomplus/storefront-renderer/functions/index.js

```diff
@@ -26,6 +28,19 @@ exports.ssr = (req, res, getCacheControl) => {
       )
   }
 
+  const proxy = url => {
+    const urlInstance = new URL(url)
+    const requestUrl = urlInstance.searchParam.get('url')
+    if (requestUrl) {
+      return axios.get(requestUrl, { headers, timeout: 3000 }, {
+        validateStatus: (status) => {
+          return status; // Resolve only if has status
```

Comment:
```suggestion
          return Boolean(status); // Resolve only if has status
```
https://axios-http.com/docs/req_config
> If `validateStatus` returns `true`